docs/proposals/2019-04-23 - Audio.md

```diff
@@ -0,0 +1,50 @@
+# Problem
+
+Currently, audio objects are not in the Content API.  We are now at a point where we would like to add audio objects into the Content API and are honing the audio object to include all of the necessary elements into the json.  Thus, I propose that embed_html, series, and audio_type be added to the json.
+
+
+# Proposal
+
+Add embed_html, series, and audio_type to audio object.
+
+
+## Embed_html
+
+HTML for the embedded audio player
+
+## Series
+
+Unique ID for podcast parent objects.  A series is a collection of related podcasts.
```

Comment:
When you say series it makes me think of an audio player that could potentially play multiple podcasts (ie show a tracks and you select which one you would like) but this configuration have an embed html for each one of them. Is that your intention? Or is the idea of a series just to provide a way to query for related audios?